docs/pages/integrating/vc.rst

```diff
@@ -68,10 +70,22 @@ Will be expanded by the node to:
       ]
     }
 
-The `visibility` property indicates the contents of the VC are published on the network, so it can be read by everyone.
+There are three parameters that can be passed:
+- `format` (optional): The format of the VC. Can be ``ldp_vc`` or ``jwt_vc``. Default is ``ldp_vc``.
+- `publishToNetwork` (optional): Whether the VC should be published on the network. Default is ``true``.
+- `visibility` (optional): The visibility of the VC. Can be ``public`` or ``private``. Default is ``private``.
+- `withStatusList2021Revocation` (optional): Whether the VC should be issued with a status list 2021 revocation. Default is ``false``.
 
-By default, the node will create credentials in JSON-LD format.
-You can specify the format by passing the `format` parameter (``jwt_vc`` or ``ldp_vc``).
+These defaults are set based on the V5 defaults. This works for the did:nuts method but might not work for other methods.
+The preferred way in V6 is to pass the following parameters:
+- `publishToNetwork`: ``false``
+- `withStatusList2021Revocation`: ``true``
```

Comment:
statuslist should only be used for long lived credentials. It's preferred to choose the expiration time such that revocation is not necessary.

Reply:
removed entire section